docs/cleos/troubleshooting.md

```diff
@@ -0,0 +1,25 @@
+## Cannot connect to RPC endpoint
+
+Check if your local nodeos is running by visiting the following URL in your browser:
```

Comment:
given that `cleos` is a command line tool, it may make more sense to provide a `curl` or `wget` one-liner to check the connectivity.